src/Servant/Client/Generic.hs

```diff
@@ -75,6 +75,7 @@ data Part
     | PQueryParams String TypeInfo
     | PRequestBody TypeInfo
     | PHeader String TypeInfo
+    | PRemoteHost
```

Comment:
I don't think `RemoteHost` need to yield a part: there is nothing useful the client-side code can do with it, so it might as well be just transparent.

Reply:
So basically it's just ```endpoints Proxy = endpoints (Proxy :: Proxy api)```?

Reply:
Yep